.github/workflows/benchmark.yml

```diff
@@ -12,35 +18,83 @@ jobs:
   
     if: | 
       startsWith(github.event.comment.body, '/benchmark') && 
-      github.event.issue.pull_request 
+      github.event.issue.pull_request || 
+      github.event_name == 'workflow_dispatch'
```

Comment:
does this associate the way you expect? Can we bracket it please, I hate trying to remember precedence.